### ISSUE TITLE
Pattern: non-editable address input example

### ISSUE THREAD

**Issue by daniellecorke** [ONSdigital/design-system#3032]
### Background/Issue

An issue was reported to the Github discussions board. A user reported that one of the address pattern examples on the DS website is not functioning correctly. 

The non-editable address input example pattern displays an error message: 

![Image](https://github.com/ONSdigital/design-system/assets/102512949/e58b850d-ec02-42c2-9d47-9f066ca78cff)

### Desired Outcome

- [x] The example works correctly. 
- [ ] The user/discussion board is updated. 

### Info
- [Design System > Patterns > Addresses](https://service-manual.ons.gov.uk/design-system/patterns/addresses)

**Comment by daniellecorke** [comment]
The example does work as intended but the documentation/explanation may need updating.

**Comment by daniellecorke** [comment]
Content design required - raise with Vicki C.

**Comment by daniellecorke** [comment]
May also need to look into advice re. ONS Address Index API.

**Comment by daniellecorke** [comment]
We need to find out who can provide access to the API.

**Comment by daniellecorke** [comment]
Once investigations into this ticket are complete and a decision has been made, #1969 needs to be updated to reflect the decision and any content design required. If no content design needed, we can close #1969.

**Comment by alessioventuriniAND** [comment]
I have been in contact with the AIMS team and the best solution is to deploy a dockerised smaller version of the address index api. 

This is the repo which contains the API https://github.com/ONSdigital/address-index-api/blob/develop/README.md#how-do-i-get-a-sneek-peak-of-the-api

We need to create a new ticket to deploy and test this API. 

We will need to add rate limiting and also only allow requests from the service manual domain. 

Once the API is deployed we will need to test it and potentially have the AIMS team look at our deployment for any suggestions. 

@daniellecorke Happy to create this ticket? 

We can leave this ticket open and update it once the work has been completed on the API deployment.

**Comment by daniellecorke** [comment]
Thanks @alessioventuriniAND - I've moved this ticket to blocked pending the completion of #3176.

**Comment by daniellecorke** [comment]
Once the address input examples are working, we can update the user and close this ticket. #1969 then needs to be unblocked and added to the UCD Board for content design.

**Comment by alessioventuriniAND** [comment]
@daniellecorke The mock address api has been deployed and the service manual preview has also been update. The examples now work. The documentation on the address input page is being updated to reflect that the api used in the examples is just a mock and any questions around the AIMS api will need to be sent to them. :)

**Comment by daniellecorke** [comment]
Thank you @alessioventuriniAND, this looks great! I will update the discussion board and users. I will also look at unblocking and reviewing the content design ticket.

**Comment by daniellecorke** [comment]
Users updated via email and discussion board.
